cmd/flowserver/flow.go

```diff
@@ -38,6 +38,7 @@ func (r *flowResponse) MarshalJSON() ([]byte, error) {
 type startRequest struct {
 	Environment *json.RawMessage       `json:"environment"`
 	Flows       json.RawMessage        `json:"flows"    validate:"required"`
+	Channels    []json.RawMessage      `json:"channels,omit_empty"`
```

Comment:
I think that makes sense - tho I think then contacts would be a list, and we'd a contact uuid somewhere else to say which contact is being started